src/components/Lightbox.tsx

```diff
@@ -92,7 +92,7 @@ type Props = {
   photos: string[]
   title?: string
   description?: string
-  content?: JSX.Element
+  content?: (currentImageIndex?: number) => JSX.Element | undefined
```

Comment:
Zie `onDelete` en `onCrop`: 
```suggestion
  content?: (imageIndex?: number) => JSX.Element | undefined
```

Reply:
Ja, beter 👍